src/LiveComponent/tests/Functional/Controller/BatchActionControllerTest.php

```diff
@@ -31,17 +31,19 @@ public function testCanBatchActions(): void
 
         $this->browser()
             ->throwExceptions()
-            ->get('/_components/with_actions', ['json' => ['props' => $dehydrated->getProps()]])
+            ->get('/_components/with_actions', ['query' => ['props' => json_encode($dehydrated->getProps())]])
```

Comment:
This is an unrelated fix, right? I mean, it was "wrong" before and was just working by some chance, right?

Reply:
Right - it was simulating `GET` request, but with data sent in body request as a JSON. I think it shouldn't be possible, but it was for some reason. Now that `POST` is handled differently it needed a proper fix :)

On the other hand - in `GET` we use `props` as a key to send all data, while for the `POST` method I used `data`. I only notices this inconsistency at the very end. But on the other hand it looks weird to me to have `props` key that contains an array with `props` method in it. Do you think it's worth to change `data` key to `props` to match query parameter name?